### ISSUE TITLE
Core dumped error

### ISSUE THREAD

**Issue by EnricoBeltramo** [microsoft/onnxruntime#5526]
**Describe the bug**
Opening a model with InferenceSession I have Segmentation fault (core dumped) 

**Urgency**
Is important part of application in an AI startup, the the project shall start within 31/10/20

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04
- ONNX Runtime installed from (source or binary): binary
- ONNX Runtime version:
onnx              1.7.0
onnxruntime       1.5.2
- Python version: 3.7
- Visual Studio version (if applicable): na
- GCC/Compiler version (if compiling from source): na
- CUDA/cuDNN version: 10
- GPU model and memory:

**To Reproduce**
- Describe steps/code to reproduce the behavior.
runnin follow python script:
import onnxruntime as ort
import numpy as np
import os
session = ort.InferenceSession('siamrpnmobilenet.onnx')
session.get_modelmeta()
first_input_name = session.get_inputs()[0].name
first_output_name = session.get_outputs()[0].name

- Attach the ONNX model to the issue (where applicable) to expedite investigation.
https://drive.google.com/file/d/10wpezlehg20MgvA054NhTb-n5eymQa1L/view?usp=sharing

**Expected behavior**
Load the model

**Screenshots**

**Additional context**
Add any other context about the problem here. If the issue is about a particular model, please share the model details as well to facilitate debugging.
The model was originally build in pytorch, and after converted in onnx format

**Comment by tracysh** [comment]
@wangyems I reproduced the crash: the crash is in ReshapeFusion::Match_One_Element_Output_Subgraph_2. The model is using opset 6 where Slice had the start/end as attributes, not inputs. The logic needs to be updated to read the attributes for older opsets (or stop matching Slice-1).

@EnricoBeltramo The model was exported from an older version of pytorch where the default ONNX opset was version 6. If optimizations are disabled, the model will still fail to run because the runtime doesn't support BatchNormalization-6. The model needs to be exported with at least opset version 7, and updating to opset version 11 would avoid the above bug. See the opset_version parameter to torch.onnx.export.

**Comment by EnricoBeltramo** [comment]
Great! Was exactly that. The original version of pytorch was 0.4. I updated to 1.6 and exported again and now it works fine. Thank you!
